dawn/src/dawn/CodeGen/Cuda-ico/CudaIcoCodeGen.cpp

```diff
@@ -1530,7 +1583,8 @@ generateF90InterfaceSI(FortranInterfaceModuleGen& fimGen,
     }
     if(includeErrorThreshold) {
       wrapper.addBodyLine(
-          "   MERGE(rel_err_threshold,DEFAULT_RELATIVE_ERROR_THRESHOLD,present(rel_err_threshold)) "
+          "   "
+          "MERGE(rel_err_threshold,DEFAULT_RELATIVE_ERROR_THRESHOLD,present(rel_err_threshold)) "
```

Comment:
Be sure to pull changes from current master, as the `MERGE` way to do select the threshold is removed there (for good reasons).

Reply:
done, thanks